digraph "CFG forio.fabric.sdk.android.services.events.EventsHandler.1.run()V" {
Node_0 [shape=record,label="{0\:\ 0x0000|MTH_ENTER_BLOCK\lSYNTHETIC\lSplitter:B:0:0x0000\l}"];
Node_1 [shape=record,label="{1\:\ ?|Catch:\{ Exception \-\> 0x0015 \}\lSplitter:B:0:0x0000\l|r3.this$0.strategy.recordEvent(r2)     \/\/ Catch:\{ Exception \-\> 0x0015 \}\l}"];
Node_2 [shape=record,label="{2\:\ 0x000b|Catch:\{ Exception \-\> 0x0015 \}\l|if (r3 == false) goto L_?     \/\/ Catch:\{ Exception \-\> 0x0015 \}\l}"];
Node_3 [shape=record,label="{3\:\ 0x000d|Catch:\{ Exception \-\> 0x0015 \}\l|r3.this$0.strategy.rollFileOver()     \/\/ Catch:\{ Exception \-\> 0x0015 \}\l}"];
Node_4 [shape=record,label="{4\:\ 0x0015|ExcHandler: Exception (r0v0 'e' java.lang.Exception A[CUSTOM_DECLARE])\lSplitter:B:0:0x0000\l|java.lang.Exception r0 = move\-exception\l}"];
Node_5 [shape=record,label="{5\:\ 0x0016|p018io.fabric.sdk.android.services.common.CommonUtils.logControlledError(r3.this$0.context, \"Failed to record event.\", r0)\l}"];
Node_6 [shape=record,label="{6\:\ ?|SYNTHETIC\lRETURN\lORIG_RETURN\l|return\l}"];
Node_7 [shape=record,label="{7\:\ ?|SYNTHETIC\lRETURN\l|return\l}"];
Node_8 [shape=record,label="{8\:\ ?|SYNTHETIC\lRETURN\l|return\l}"];
MethodNode[shape=record,label="{public void io.fabric.sdk.android.services.events.EventsHandler.1.run((r3v0 'this' io.fabric.sdk.android.services.events.EventsHandler$1 A[IMMUTABLE_TYPE, THIS])) }"];
MethodNode -> Node_0;
Node_0 -> Node_1;
Node_0 -> Node_4;
Node_1 -> Node_2;
Node_2 -> Node_3[style=dashed];
Node_2 -> Node_7;
Node_3 -> Node_8;
Node_4 -> Node_5;
Node_5 -> Node_6;
}

